frontend/src/metabase-lib/queries/StructuredQuery.ts

```diff
@@ -1192,7 +1196,7 @@ class StructuredQueryInner extends AtomicQuery {
   fieldsOptions(
     dimensionFilter: DimensionFilterFn = dimension => true,
   ): DimensionOptions {
-    if (this.isBareRows() && !this.hasBreakouts()) {
+    if (this.isBareRows() && !this.hasAnyBreakouts()) {
```

Comment:
This line did not change semantically but behavior did.
It's not possible to guess/tell the difference between `hasBreakouts()` and `hasAnyBreakouts()` without looking at their code. I suggest to make the name more specific:


```suggestion
    if (this.isBareRows() && !this.anySourceQueryHasBreakouts()) {
```